frontend/src/components/JourneysContainer.tsx

```diff
@@ -0,0 +1,151 @@
+import {
+  IonButton,
+  IonCard,
+  IonCardHeader,
+  IonCardSubtitle,
+  IonCardTitle,
+  IonInput,
+  IonItem,
+  IonLabel,
+  IonList,
+  IonTextarea
+} from "@ionic/react";
+import { Journeys, Location, Station, Stop } from "hafas-client";
```

Comment:
The usage of types from a backend dependency is due to the problem with the return type definitions you mentioned earlier, right? Can you add to #157 that when it's done this dependency should also be removed from the frontend?

Reply:
I haven´t had that in mind, but you´re right: Once the definition of API return types is working correctly the '@types/hafas-client' dependency in frontend can (and should) be removed.